content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -18,19 +17,38 @@ Kubernetes DNS schedules a DNS Pod and Service on the cluster, and configures
 the kubelets to tell individual containers to use the DNS Service's IP to
 resolve DNS names.
 
-### What things get DNS names?
-
 Every Service defined in the cluster (including the DNS server itself) is
-assigned a DNS name.  By default, a client Pod's DNS search list will
-include the Pod's own namespace and the cluster's default domain.  This is best
-illustrated by example:
+assigned a DNS name. By default, a client Pod's DNS search list will
+include the Pod's own namespace and the cluster's default domain. 
+
+### Namespaces of Services 
+
+DNS queries for services change based on namespace of the pod. DNS queries that don't specifiy a namespace are limited to the pod's namespace. Services in other namespaces may be accessed by specifying the namespace. 
```

Comment:
Please wrap the long lines for the ease of change tracking in the future.
Please use active voice when possible.

Reply:
is there a tool you use for auto wrapping long lines?

Reply:
I'm using VIM. Auto-wrapping or reformatting a paragraph can be easily done by simply press 'Q' in a paragraph.

Reply:
ah thank you!